build/gulpfile.vscode.linux.js

```diff
@@ -15,12 +15,31 @@ const util = require('./lib/util');
 const packageJson = require('../package.json');
 const product = require('../product.json');
 
+const linuxPackageRevision = Math.floor(new Date().getTime() / 1000);
+
+const flatpakManifest = {
+	appId: 'com.microsoft.CodeOSS',
+	sdk: 'org.freedesktop.Sdk',
+	runtime: 'org.freedesktop.Sdk',
+	runtimeVersion: '1.4',
+	base: 'io.atom.electron.BaseApp',
+	baseFlatpakref: 'https://s3-us-west-2.amazonaws.com/electron-flatpak.endlessm.com/electron-base-app-master.flatpakref',
+	command: "code-oss",
+	symlinks: [
+		['/share/code-oss/code-oss', '/bin/code-oss'],
+	],
+	finishArgs: [
+		'--share=ipc', '--socket=x11',  // Allow showing X11 windows.
+		'--filesystem=home',            // Allow access to home directory.
+		'--device=dri',                 // Allow OpenGL rendering.
```

Comment:
I think we can add all of these:

* Indeed `--share=network` is a good idea, somehow I missed the fact that the application goes online to fetch extensions. Not to mention that it may want to push/pull from Git repositories!
* As for notifications, VSCode itself does not use them, and checking the [popular extensions page](https://marketplace.visualstudio.com/search?target=VSCode&sortBy=Downloads) I do not see how any of them would like to use them, and [searching for “notification”](https://marketplace.visualstudio.com/search?term=notification&target=VSCode&sortBy=Relevance) brings up zero results... but anyway it's a small allowance, but it doesn't hurt having it.
* Funnily enough, there's [a few extensions](https://marketplace.visualstudio.com/search?term=sound&target=VSCode&sortBy=Relevance) which use audio devices, so let's add that, too!